### ISSUE TITLE
Enhancement: Radio Group - Add color property

### ISSUE THREAD

**Issue by asangma** [Esri/calcite-components#477]
Similar to button, it could be useful to have the color property on RadioGroup.

I see cases where there might be a radio group in a panel near a "primary" and "secondary"  button set, e.g. `Done` `Cancel`, and the RadioGroup starts to look really similar.

UNTZ UNTZ UNTZ

**Comment by macandcheese** [comment]
Yeah, can see that use case.

Would using the "appearance" nomenclature from `calcite-button` work here (solid, outline) vs "color"?

**Comment by asangma** [comment]
I'm cool with whatever nomenclature you prefer. 
But I was thinking kinna lik-a-dis.
![Screen Shot 2020-04-16 at 8 48 56 AM](https://user-images.githubusercontent.com/12503298/79477829-8a0fe500-7fbf-11ea-8c2e-aacea3f1afe2.png)


Note, I'm using a gray that's more like `6a6a6a`. The `404040` that `button color="dark"` uses seems too strong in this context. I think even the `6a6a6a` is a bit strong for the border in this case. But I'm open.

**Comment by macandcheese** [comment]
@bstifle any thoughts here on a new Radio Button style?

**Comment by bstifle** [comment]
Not a fan of the gray tbh. if not interacting with it readily, its weird which one is selected since there are only 2. @asangma @macandcheese

**Comment by bstifle** [comment]
![image](https://user-images.githubusercontent.com/14083530/79486848-293ad980-7fcc-11ea-95e7-1a5ca921ecde.png)


maybe something more like this? (even if the highlight were gray) ((would prefer blue but ¯\_(ツ)_/¯))

https://www.esri.com/en-us/arcgis/products/arcgis-desktop/buy

**Comment by paulcpederson** [comment]
We use that blue outline style _a lot_ in ArcGIS Online. I like that a lot better than the gray. My vote is @bstifle 's above take

**Comment by paulcpederson** [comment]
Can use the same prop nomenclature as button as well so it's an easy and consistent api

**Comment by macandcheese** [comment]
Yep, `appearance="solid/outline"` would match button, default to solid.

**Comment by asangma** [comment]
Thanks y'all. I'll try it out.

**Comment by asangma** [comment]
![image](https://user-images.githubusercontent.com/12503298/79620311-5bc7fd80-80c4-11ea-8e07-ccb8d7946133.png)

Y'all are so good! Ship it!

**Comment by mitc7862** [comment]
Looks good. Like the blue outline version.

**Comment by julio8a** [comment]
Looks really good!
